src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1098,13 +1098,12 @@ private stmt_ ::= expr_stmt
 // Utils
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
-GTGTEQ ::= '>' '>' '='
-GTGT   ::= '>' '>'
-GTEQ   ::= '>' '='
-
-LTLTEQ ::= '<' '<' '='
-LTLT   ::= '<' '<'
-LTEQ   ::= '<' '='
+external gtgteq ::= collapse GTGTEQ GT GT EQ
```

Comment:
Actually this cannot be done. When I change `GTGTEQ` to `'>>='`, `collapse` is provided with a `Parser` object instead of `IElementType`.